docs/core-concepts/dnx-migration.md

```diff
@@ -131,6 +142,10 @@ brought, the framework needs to be one of the following:
 1. `netcoreapp1.0` - if you are writing applications on .NET Core (including ASP.NET Core applications)
 2. `netstandard1.5` - if you are writing class libraries for .NET Core
 
+If you are using other `dnx` targets, like `dnx451` you will need to change those as well. `dnx451` should go to `net451`. 
```

Comment:
nite: "`dnx451` should go to `net451`" ---> "`dnx451` should be changed to `net451`"